src/rust/sysmon-subgraph-generator/src/main.rs

```diff
@@ -699,337 +638,27 @@ where
-
-    let sysmon_graph_generator_queue_url = std::env::var("SYSMON_GRAPH_GENERATOR_QUEUE_URL")
-        .expect("SYSMON_GRAPH_GENERATOR_QUEUE_URL");
-    local_sqs_service(
-        sysmon_graph_generator_queue_url,
-        "local-grapl-unid-subgraphs-generated-bucket",
-        Context {
-            deadline: Utc::now().timestamp_millis() + 10_000,
-            ..Default::default()
-        },
-        init_s3_client(),
-        init_sqs_client(),
-        ZstdDecoder::default(),
-        SubgraphSerializer {
-            proto: Vec::with_capacity(1024),
-        },
-        generator,
-        NopCache {},
-        |_, event_result| {
-            dbg!(event_result);
-        },
-        move |bucket, key| async move {
-            let output_event = S3Event {
-                records: vec![S3EventRecord {
-                    event_version: None,
-                    event_source: None,
-                    aws_region: None,
-                    event_time: chrono::Utc::now(),
-                    event_name: None,
-                    principal_id: S3UserIdentity { principal_id: None },
-                    request_parameters: S3RequestParameters {
-                        source_ip_address: None,
-                    },
-                    response_elements: Default::default(),
-                    s3: S3Entity {
-                        schema_version: None,
-                        configuration_id: None,
-                        bucket: S3Bucket {
-                            name: Some(bucket),
-                            owner_identity: S3UserIdentity { principal_id: None },
-                            arn: None,
-                        },
-                        object: S3Object {
-                            key: Some(key),
-                            size: 0,
-                            url_decoded_key: None,
-                            version_id: None,
-                            e_tag: None,
-                            sequencer: None,
-                        },
-                    },
-                }],
-            };
-
-            let sqs_client = init_sqs_client();
-
-            // publish to SQS
-            sqs_client
-                .send_message(SendMessageRequest {
-                    message_body: serde_json::to_string(&output_event)
-                        .expect("failed to encode s3 event"),
-                    queue_url: std::env::var("NODE_IDENTIFIER_QUEUE_URL")
-                        .expect("NODE_IDENTIFIER_QUEUE_URL"),
-                    ..Default::default()
-                })
-                .await?;
-
-            Ok(())
-        },
-    )
-    .await?;
-    Ok(())
-}
-
-fn main() -> Result<(), Box<dyn std::error::Error>> {
-    simple_logger::init_with_level(grapl_config::grapl_log_level())
-        .expect("Failed to initialize logger");
-
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    simple_logger::init_with_level(log::Level::Info).unwrap();
```

Comment:
should probably use `grapl_config::grapl_log_level()` here

Reply:
I'll patch now.